tools/test/examples/examples_lib.py

```diff
@@ -349,7 +359,15 @@ def status(message):
     return results
 
 
-def compile_repos(config, toolchains, targets, profile, verbose, examples):
+def compile_repos(
+    config,
+    toolchains,
+    targets,
+    profiles,
+    verbose,
+    examples,
+    post_checks,
```

Comment:
Can the styling change be in separate commit ? I would even send it as separate PR, this distract us from functional changes in this PR

Reply:
https://github.com/ARMmbed/mbed-os/pull/11429/commits/c1f15a4cb0fa2eea9a9aa7a5ed9bd640cc5fa9de